tests/accessibility/getAccessibilityTree.test.ts

```diff
@@ -139,6 +139,76 @@ test(
   }),
 );
 
+test(
+  'role="presentation"',
+  withBrowser(async ({ utils, page }) => {
+    const body = (await page.$('body'))!;
+    await utils.injectHTML(`<h1 role="presentation">Sample Content</h1>`);
+    // Role="presentation" and role="none" are equivalent
+    // They make it as if the outer element wasn't there.
+    expect(await getAccessibilityTree(body)).toMatchInlineSnapshot(
+      `text "Sample Content"`,
+    );
+    await utils.injectHTML(`<h1 role="none">Sample Content</h1>`);
+    // Role="presentation" and role="none" are equivalent
+    // They make it as if the outer element wasn't there.
+    expect(await getAccessibilityTree(body)).toMatchInlineSnapshot(
+      `text "Sample Content"`,
+    );
+
+    // The li (role=listitem) children are required owned elements of the ul (role=list)
+    // so when the list is set to role=presentation, the required owned elements are too
+    // The third li has a different role so it is not a required owned element of the list
```

Comment:
This read a bit funny. I'm not sure my suggestion is better. 🤔 
```suggestion
    // The li (role=listitem) children are required owned elements of the ul (role=list)
    // When the list is set to role=presentation, the required owned elements are also
    // The third li has a different role so it is not a required owned element of the list
```

Reply:
> // The third li has a different role so it is not a required owned element of the list

@calebeby I'm trying to find where this expectation is described in the [document you shared](https://www.digitala11y.com/presentation-role/) but I can't find it. Can you direct me to where this is documented? I simply want to confirm/understand. Thanks! 😉

Reply:
In the spec it says that the parent role="presentation" should only cascade to accessibility tree children which are required owned elements of the parent element and which have no explicit role defined.

> When an explicit or inherited role of presentation is applied to an element with the implicit semantic of a WAI-ARIA role that has required owned elements, in addition to the element with the explicit role of presentation, the user agent MUST apply an inherited role of presentation to any owned elements that do not have an explicit role defined. Also, when an explicit or inherited role of presentation is applied to a host language element which has required children as defined by the host language specification, in addition to the element with the explicit role of presentation, the user agent MUST apply an inherited role of presentation to any required children that do not have an explicit role defined.

I'm realizing that I should probably add to this test a check for an explicit role which is the same as the implicit role `<li role="listitem">` (I think the correct behavior is for the role="presentation") to not cascade in that case.

Reply:
Confirmed this in Chrome/FF:

```html
<ul role="presentation">
	  <li>Sample Content</li>
	  <li>More Sample Content</li>
	  <li role="listitem">Hi</li>
</ul>
```

When the `<ul>` has role="presentation":

<img width="429" alt="Screen Shot 2021-12-07 at 3 29 07 PM" src="https://user-images.githubusercontent.com/13206945/145121839-784fe415-00c0-40e3-93b7-207af1757e50.png">

When the parent has no explicit role:

<img width="429" alt="Screen Shot 2021-12-07 at 3 31 23 PM" src="https://user-images.githubusercontent.com/13206945/145122020-7316f92e-8f24-4486-a581-8359c3d4eac6.png">

Reply:
I pushed a fix and a test to handle this edge case where the explicit role set on a descendant makes it a required owned element, but the presence of the explicitly set role means that the role="presentation" on the parent does not cascade to it.

Reply:
@gerardo-rodriguez re: your initial suggested change in the comment: I rephrased that sentence so I think it reads better now. Let me know what you think!